apps/Gateway/conf/nginx.conf

```diff
@@ -0,0 +1,81 @@
+user nginx;
+worker_processes 1;
+
+events {
+    worker_connections 1024;
+}
+
+http {
+    include       mime.types;
+    default_type  application/octet-stream;
+
+    sendfile        on;
+    keepalive_timeout  65;
+	
+    server {
+        listen       80;
+        server_name  localhost;
+
+        location /api {
```

Comment:
Tego raczej nie powinno być, nie mamy ścieżki /api do żadnego serwisu, a w linku to był tylko przykład. Cors powinien być ustawiony na poziomie każdego z przekierowań lub w ogóle na poziomie serwera o ile się da